test/cipher_opt.js

```diff
@@ -45,7 +47,14 @@ describe('CipherOpt', () => {
   });
 
   it('asymmetrically encrypts data to be written to immutable structure', async () => {
-    const differentApp = await h.createAltAuthTestApp();
+    let differentApp = autoref(new App({
```

Comment:
How about changing the `createTestApp` function to accept a `partialAppInfo` param instead of a `scope` param, which then merges it (`Object.assign(appInfo, partialAppInfo)`) with the default `appInfo`, by doing this we can call it from here with the modified appInfo we need here?